include/exiv2/bmffimage.hpp

```diff
@@ -0,0 +1,170 @@
+// ***************************************************************** -*- C++ -*-
+/*
+ * Copyright (C) 2021 Exiv2 authors
+ * This program is part of the Exiv2 distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, 5th Floor, Boston, MA 02110-1301 USA.
+ */
+
+#pragma once
+
+// *****************************************************************************
+#include "exiv2lib_export.h"
+
+// included header files
+#include "image.hpp"
+#include "iostream"
+
+// *****************************************************************************
+// namespace extensions
+namespace Exiv2
+{
+    EXIV2API bool enableBMFF(bool enable = true);
```

Comment:
Should not this function has return type `void` ? I see that in the examples and documentation we do not use the returned value.

Reply:
The return value is important.  It tells the caller if the code has been built with BMFF support.  In the documentation and samples, we ignore the return value.  However, "real" applications can use this to "gray out" a UI option such as "Use bmff support", or to post an alert such as _bmff support cannot be enabled on this platform_.